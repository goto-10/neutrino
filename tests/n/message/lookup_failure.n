# Copyright 2014 the Neutrino authors (see AUTHORS).
# Licensed under the Apache License, Version 2.0 (see LICENSE).

def $foo();
def $level2() => $foo(null);
def $level1() => $level2();

do $level1();
