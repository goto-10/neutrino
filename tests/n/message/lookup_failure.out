Error: %<condition: Signal(escape)>
Info: --- backtrace ---
- :foo(null)
- :level2()
- :level1()
